tests/integration/api_container_test.py

```diff
@@ -422,6 +422,23 @@ def test_create_with_stop_timeout(self):
         config = self.client.inspect_container(container)
         assert config['Config']['StopTimeout'] == 25
 
+    @requires_api_version('1.26')
+    def test_create_with_init(self):
+        ctnr = self.client.create_container(
+            BUSYBOX, 'true',
+            host_config=self.client.create_host_config(
+                init='true'
```

Comment:
'true' is not a valid boolean. This test would fail, but the decorator prevents it from being run in the first place.